### PR TITLE
Release/1.5.1

### LINKED ISSUE #26: Release version 1.5.1
This issue is for tracking changes for the 1.5.1 release. Target release date: **October 2023**.

### [Release steps](https://github.com/10up/block-catalog/blob/develop/CONTRIBUTING.md#release-instructions)

- [x] Branch: Starting from `develop`, cut a release branch named `release/1.5.1` for your changes.
- [x] Version bump: Bump the version number in `block-catalog.php`, `package-lock.json`, `package.json`, and `readme.txt` if it does not already reflect the version being released. In `block-catalog.php` update both the plugin "Version:" property and the plugin `BLOCK_CATALOG_PLUGIN_VERSION` constant.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` file with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.gitattributes`.
- [x] Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk`, ensuring you pull the most recent changes into `develop` first (`git checkout develop && git pull origin develop && git checkout trunk && git merge --no-ff develop`). `trunk` contains the stable development version.
- [x] Push: Push your trunk branch to GitHub (e.g. `git push origin trunk`).
- [x] [Compare](https://github.com/10up/block-catalog/compare/trunk...develop) `trunk` to `develop` to ensure no additional changes were missed.
- [x] Test the pre-release ZIP locally by [downloading](https://github.com/10up/block-catalog/actions/workflows/build-release-zip.yml) it from the Build release zip action artifact and installing it locally. Ensure this zip has all the files we expect, that it installs and activates correctly and that all basic functionality is working.
- [x] Release: Create a [new release](https://github.com/10up/block-catalog/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch. Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the closed issues on the [milestone](https://github.com/10up/block-catalog/milestone/4?closed=1).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/block-catalog/actions/workflows/wordpress-plugin-deploy.yml) to finish deploying to the WordPress.org repository. If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on [WordPress.org](https://wordpress.org/plugins/block-catalog/). This may take a few minutes.
- [x] Close milestone: Edit the [milestone](https://github.com/10up/block-catalog/milestone/4) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description field`), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.5.1` do not make it into the release, update their milestone to `1.5.2`, `1.6.0` or `Future Release`.

### PR DESCRIPTION
**Pull request by dkotter**
### Description of the Change

Version bump and prep for the 1.5.1 release

Closes #26 

### How to test the Change

Verify all file changes look accurate

### CODE REVIEW

File: .github/workflows/wordpress-plugin-deploy.yml [hunk 1]
```diff
@@ -1,18 +1,41 @@
 name: Deploy to WordPress.org
```
Comment (dkotter) [inline .github/workflows/wordpress-plugin-deploy.yml:1]: Not exactly sure how this plugin has been deployed to WordPress.org up until this point (doesn't seem like this Action was being used and I see there's a few bin scripts around building a release asset). I think the changes I've made here will make for a successful deploy but could use another eye on that

Comment (jeffpaul) [reply .github/workflows/wordpress-plugin-deploy.yml:1]: @dsawardekar I believe you've done manual deploys for this, so hoping you can help comment/review these changes?

**Review by jeffpaul — DISMISSED** [review]
Looks good, thanks for handling the release!

File: .github/workflows/wordpress-plugin-deploy.yml (hunk 1 above)
Comment (dsawardekar) [reply .github/workflows/wordpress-plugin-deploy.yml:1]: @dkotter I was pushing to the .org SVN repo to deploy. The automatted deployment looks much better than the manual one. Let me know once released, and I can test on a few sites that I'm currently using it on.

**Review by dsawardekar — DISMISSED** [review]

**Pull request merged**